package.json

```diff
@@ -64,6 +64,8 @@
 		"@wordpress/wordcount": "file:packages/wordcount"
 	},
 	"devDependencies": {
+		"@actions/core": "1.0.0",
+		"@actions/github": "1.0.0",
```

Comment:
Hmm, I wouldn't have thought these dependencies were needed in the root package.json.

Reply:
According to https://github.com/WordPress/gutenberg/blob/4665d7074dfeae43eac02b1d2e4ad5713832078e/packages/README.md#development-dependencies it's intended that way.

Reply:
Thanks for the pointer @swissspidy.

Reply:
But these are also listed as `dependencies` in `packages/project-management-automation/package.json` . Surely they don't need to be declared twice? They seem more appropriate as `dependencies` to me than as `devDependencies`, since they're required for the runtime script of the module.